### PR TITLE
Changed default umask value from 0777 to 022 #9916

### PR DESCRIPTION
**Pull request by bkeyes93**

### CODE REVIEW

**Comment by sbc100** [comment]
I guess this fixes #9916?

**Comment by bkeyes93** [comment]
Yes. This is my first time submitting an actual PR, so hopefully I did it correctly.

File: src/library_syscall.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ var SyscallsLibrary = {
 
     // global state
     mappings: {},
-    umask: 0x1FF,  // S_IRWXU | S_IRWXG | S_IRWXO
```
Comment (sbc100) [inline src/library_syscall.js]: I think JS supports octal values so could instead just write O22 here which might be more readable.

I'm surprised this didn't effect any tests.  Might be worth adding one?

Comment (bkeyes93) [reply src/library_syscall.js]: Okay, I changed it to octal representation using the modern syntax for octals. I'm not sure how to add a test for this though.

**Comment by sbc100** [comment]
Can you add yourself to AUTHORS?

**Comment by bkeyes93** [comment]
Okay, done.

File: src/library_syscall.js (hunk 1 above)
Comment (kripken) [reply src/library_syscall.js]: Possibly one of the tests in `tests/unistd/` is a natural place to add a test for this. Perhaps `misc.c`?

**Comment by sbc100** [comment]
Hmm.. renaming the master branch for some reason closed this PR.. i thought it was just supposed to re-target open PR.  Feel free to re-open/re-create.

**Pull request closed without merging**